src/compiler/crystal/macros.cr

```diff
@@ -1187,6 +1187,17 @@ module Crystal::Macros
   class Out < UnaryExpression
   end
 
+  # An `offsetof` expression.
+  class OffsetOf < ASTNode
+    # Returns the type that has been used in this `offsetof` expression.
+    def structure : TypeNode
+    end
+
+    # Returns the name of the instance variable used in this `offsetof` expression as a MacroId.
+    def member : MacroId
```

Comment:
We're using the name `instance_variable` in other places, so this should follow.